hack/tests/origin-conformance.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -ex
+
+PATH=`go env GOPATH`/bin::$PATH
+TEST_SUITE="${TEST_SUITE:-openshift/conformance/parallel/minimal}"
+
+cleanup() {
+  set +e
+
+  generate_artifacts
+
+  delete
+}
+
+fetch_origin() {
+  local branch="$1"
+  local remote="https://github.com/openshift/origin.git"
+
+  test -d /tmp/origin || git clone --depth=1 --branch=${branch} $remote /tmp/origin
+}
+
+install_binaries() {
+  # install ginkgo
+  go get -v github.com/onsi/ginkgo/ginkgo
```

Comment:
can we get away with installing ginkgo in the `ci-base` image instead? https://github.com/openshift/release/blob/master/projects/azure/base-images/ci-base.yaml

Reply:
I thought about it, then thought what could happen if you need a specific version of ginkgo installed? granted, my argument is weak. I will add it to the PR.

Reply:
We're not pinning here to a specific version of ginkgo so it seems safe to do it in the `ci-base` image. I think you've made that change already so this can be removed :)